data/src/main/java/me/androidbox/data/local/dao/ReminderDao.kt

```diff
@@ -0,0 +1,25 @@
+package me.androidbox.data.local.dao
+
+import androidx.room.*
+import kotlinx.coroutines.flow.Flow
+import me.androidbox.data.local.DatabaseConstant
+import me.androidbox.data.local.entity.ReminderEntity
+
+@Dao
+interface ReminderDao {
+    @Query("SELECT * FROM ${DatabaseConstant.REMINDER_TABLE}")
+    fun getReminder(): Flow<List<ReminderEntity>>
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    fun insertReminder(reminderEntity: ReminderEntity)
+
+    /**
+     * TODO Delete only a single reminder
+     * */
+    @Delete
+    fun deleteReminder(reminderEntity: ReminderEntity)
```

Comment:
Is this correct that we only want to delete a single reminder

Reply:
Yes, same comment as for events